src/lib/npfg/CourseToAirspeedRefMapper.cpp

```diff
@@ -36,42 +36,49 @@ using matrix::Vector2f;
 
 float
 CourseToAirspeedRefMapper::mapCourseSetpointToHeadingSetpoint(const float bearing_setpoint, const Vector2f &wind_vel,
-		float airspeed_true) const
+		float airspeed_sp) const
 {
-
-	Vector2f bearing_vector = Vector2f{cosf(bearing_setpoint), sinf(bearing_setpoint)};
+	const Vector2f bearing_vector = Vector2f{cosf(bearing_setpoint), sinf(bearing_setpoint)};
 	const float wind_cross_bearing = wind_vel.cross(bearing_vector);
-	const float wind_dot_bearing = wind_vel.dot(bearing_vector);
 
-	const Vector2f air_vel_ref = refAirVelocity(wind_vel, bearing_vector, wind_cross_bearing,
```

Comment:
@sfuhrer refAirVelocity still exists in the header

Reply:
removed